cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -241,6 +245,18 @@
     {% for feature, sources, targets in feature_reos %}
         {% if forloop.first %}
             <div style="display: flex; justify-content: space-between">
+                {% if profile_header_feature_type != "gene" and profile_header_feature_type != "transcript" and profile_header_feature_type != "exon" %}
+                <h1>
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
+                        <!--! Font Awesome Pro 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
+                        <path
+                            d="M416 0c17.7 0 32 14.3 32 32c0 59.8-30.3 107.5-69.4 146.6c-28 28-62.5 53.5-97.3 77.4l-2.5 1.7c-11.9 8.1-23.8 16.1-35.5 23.9l0 0 0 0 0 0-1.6 1c-6 4-11.9 7.9-17.8 11.9c-20.9 14-40.8 27.7-59.3 41.5H283.3c-9.8-7.4-20.1-14.7-30.7-22.1l7-4.7 3-2c15.1-10.1 30.9-20.6 46.7-31.6c25 18.1 48.9 37.3 69.4 57.7C417.7 372.5 448 420.2 448 480c0 17.7-14.3 32-32 32s-32-14.3-32-32H64c0 17.7-14.3 32-32 32s-32-14.3-32-32c0-59.8 30.3-107.5 69.4-146.6c28-28 62.5-53.5 97.3-77.4c-34.8-23.9-69.3-49.3-97.3-77.4C30.3 139.5 0 91.8 0 32C0 14.3 14.3 0 32 0S64 14.3 64 32H384c0-17.7 14.3-32 32-32zM338.6 384H109.4c-10.1 10.6-18.6 21.3-25.5 32H364.1c-6.8-10.7-15.3-21.4-25.5-32zM109.4 128H338.6c10.1-10.7 18.6-21.3 25.5-32H83.9c6.8 10.7 15.3 21.3 25.5 32zm55.4 48c18.4 13.8 38.4 27.5 59.3 41.5c20.9-14 40.8-27.7 59.3-41.5H164.7z"
+                        />
+                    </svg>
+                    {{ feature.get_feature_type_display }}
```

Comment:
I think we talked about on `[feature type] at [location]`

Reply:
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/250afe32-99b3-49b6-80dc-f4c22e7127cc)
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/8a0e3cda-28d0-489a-a42d-d090b5200df2)

Feature type and location is here. Location is always understood for us in clinical etc --if what you are referring to is the "at".

Reply:
Okay, that looks good.